harvester/DBInterface.py

```diff
@@ -580,7 +580,7 @@ def write_record(self, record, repo):
                 elif existing_record["local_identifier"] != record["identifier"]:
                     modified_upstream = True
                 try:
-                    if existing_record["files_size"] != record["files_size"]:
+                    if existing_record["files_size"] != record.get("files_size"):
                         record["files_altered"] = 1
                         modified_upstream = True
                 except AttributeError:
```

Comment:
Did we decide what to do about this exception handling? I think we could either use a larger try/except block, or leave this out entirely (since it is already handled at a higher level), rather than only catching this specific exception

Reply:
We should probably have a slightly higher up try/except block to catch all these potential AttibuteErrors, rather than this one specific try/except block